tests/libtransmission/makemeta-test.cc

```diff
@@ -234,19 +234,18 @@ TEST_F(MakemetaTest, announceSingleTracker)
 
     // generate the torrent and parse it as a variant
     EXPECT_EQ(nullptr, builder.make_checksums().get());
-    auto top = tr_variant{};
-    auto const benc = builder.benc();
-    EXPECT_TRUE(tr_variantFromBuf(&top, TR_VARIANT_PARSE_BENC | TR_VARIANT_PARSE_INPLACE, benc));
+    auto top = tr_variant_serde::benc().parse(builder.benc());
```

Comment:
ditto

Reply:
I don't think that's correct --the string returned by `builder.benc()` goes out-of-scope after the parsing is done